styles/inoreader/catppuccin.user.css

```diff
@@ -69,309 +69,485 @@
+  .search_context_dropdown.open .search_context_filter.active,
+  .nav.nav-hover .active-url,
+  .lang_bubble.lang_bubble_current {
+    background-color: @surface1;
+    color: @accent-color;
+  }
 
-    .catalog_follow_featured_collection,
-    .catalog_sub_section_title,
-    .search_feed_wrapper {
-      border-color: @surface0;
-    }
+  .btn.btn-outline-text {
+    color: @text;
+    border-color: @subtext0;
+  }
 
-    .pricing_teams_hero {
-      background-color: @surface0;
-      background-image: none;
-    }
+  .btn.btn-primary,
+  .btn.btn-primary:hover {
+    color: @text;
+    background-color: @accent-color;
+    border-color: @accent-color;
+  }
 
-    .parent_div_inner:hover,
-    .inno_toolbar_button_menu_item:hover,
-    .preferences_profile_wrapper,
-    code,
-    pre {
-      background-color: @surface0;
-      color: @text;
-    }
+  .dropdown-menu-active,
+  .btn.btn-outline-text:hover {
+    color: @text;
+    border-color: @subtext0;
+    background-color: @surface0;
+  }
 
-    .preferences_profile_plan_wrapper,
-    .st_subheader,
-    .catalog_sub_section_tab.catalog_sub_section_tab_current,
-    .catalog_sub_section_tab:hover {
-      background: @surface1;
-      color: @text;
-    }
 
-    .article_footer .article_footer_main_buttons .article_footer_buttons:hover,
-    .article_footer
-      .article_footer_main_buttons
-      .article_footer_buttons_current,
-    .article_footer .icon-article_topbar_more_menu:hover,
-    .st_header,
-    input[type="radio"].filter_radio:checked + label,
-    input[type="checkbox"].filter_radio:checked + label {
-      background-color: @surface2;
-      color: @text;
-    }
+  input[type="text"]:focus,
+  input[type="checkbox"]:focus + label,
+  input[type="checkbox"]:active + label,
+  input[type="button"]:focus,
+  input[type="button"]:active,
+  textarea:focus,
+  select:focus,
+  button:focus,
+  input[type="password"]:focus,
+  input[type="email"]:focus {
+    box-shadow: 0 0 3px @accent-color !important;
+    border-color: @accent-color !important;
+  }
 
-    .normal_dim {
-      color: @subtext1;
-    }
+  input[type="text"],
+  input[type="number"],
+  input[type="password"],
+  select,
+  textarea {
+    background-color: @base !important;
+    color: @text !important;
+    border-color: @subtext0;
+  }
 
-    .parent_div_inner_selected,
-    #preferences_header,
-    .add_content_simulated_button_menu_line_item:hover {
-      background-color: @accent-color;
-      color: @mantle;
-    }
+  .graylink_darker,
+  .article_short_contents {
+    color: @subtext0;
+  }
 
-    #preferences_header .icon16,
-    #preferences_dialog_close_button span.icon16 {
-      color: @mantle;
-    }
+  div.article_magazine.article_current,
+  div.article_magazine.article_bulk_selected {
+    box-shadow: 0 0 0 1px @subtext0;
+  }
+  .view_style_2 #reader_pane .ar.article_current_3way,
+  .view_style_2 #reader_pane .ar.article_expanded {
+    box-shadow: none;
+    border: 1px solid @text;
```

Comment:
we dont want a opinionated change

Reply:
to build on this. consider using `border-color` instead.

Reply:
> we dont want a opinionated change

The linter will catch issues like this. No need to comment on them Omar.